### ISSUE TITLE
Pypandoc can not be used if $HOME is not set

### ISSUE THREAD

**Issue by tkob** [bebraw/pypandoc#103]
Pandoc and any functions of pypandoc fail is $HOME environment variable is not set:

```
>>> import os, pypandoc
>>> os.unsetenv('HOME')
>>> pypandoc.get_pandoc_version()
pandoc: HOME: getAppUserDataDirectory: does not exist (no environment variable)
Maybe try:

    sudo yum install pandoc
See http://johnmacfarlane.net/pandoc/installing.html
for installation options
---------------------------------------------------------------

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/pypandoc/__init__.py", line 251, in get_pandoc_version
    _ensure_pandoc_path()
  File "/usr/lib/python2.7/site-packages/pypandoc/__init__.py", line 336, in _ensure_pandoc_path
    raise OSError("No pandoc was found: either install pandoc and add it\n"
OSError: No pandoc was found: either install pandoc and add it
to your PATH or install pypandoc wheels with included pandoc.
```

As the first line of the output, which comes from stderr, shows, `pandoc --version` tries to look up $HOME (to show default data directory) and fails because it is not set.

This is undesirable when pypandoc is used from a user with no $HOME, such as `apache` user.

A workaround is to use `os.putenv`, but the effect is somewhat non-local. Better solution would to utlilize `--data-dir` option of  `pandoc` command.

**Comment by bebraw** [comment]
A `--data-dir` based PR would be welcome. Ideally the change should be backwards compatible.

**Comment by janschulz** [comment]
Closed in #104
